src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$header-height: 60px;
+$card-width: 200px;
+
+$c-main-text: #060b35;
+$c-secondary-text: #616070;
+$c-blue-accent: #00acdc;
+$c-white-accent: #fff;
```

Do not abbreviate names to unreadable ones.
```suggestion
$main-text-color: #060b35;
$secondary-text-color: #616070;
$blue-accent-color: #00acdc;
$white-accent-color: #fff;
```